__device-tests__/gutenberg-editor-sanity-test-1-visual.test.js

```diff
@@ -331,5 +331,54 @@ describe( 'Gutenberg Editor - Test Suite 1', () => {
 			);
 			expect( screenshot ).toMatchImageSnapshot();
 		} );
+
+		it( 'should display the block outline of a block inserted from the appender', async () => {
+			await editorPage.initializeEditor();
+
+			await editorPage.addNewBlock( blockNames.socialIcons );
+
+			const appenderButton = isAndroid()
+				? await editorPage.waitForElementToBeDisplayedByXPath(
+						`//android.widget.Button[@resource-id="appender-button"]`
+				  )
+				: await this.waitForElementToBeDisplayedById(
+						'appender-button'
+				  );
+			await appenderButton.click();
```

Comment:
I did note the existence of the [addBlockUsingAppender](https://github.com/wordpress-mobile/gutenberg-mobile/blob/3d08d6321b84d1a6115b82bf79a0bd03503fb5db/__device-tests__/gutenberg-editor-sanity-test-4-visual.test.js#L313) helper, and attempted to use it, but ran into issues with passing the Amazon block name as the second argument to that function. (I think perhaps it is not registered via `blockNames` in the same way for other primary blocks like paragraph.) Either way, selecting the block via XPath here works, but I'm interested in feedback if this could have worked with `addBlockUsingAppender` instead.